src/interfaces.ts

```diff
@@ -122,41 +122,22 @@ export enum DefaultEditorId {
   'css' = 'styles.css',
 }
 
-export type CustomEditorId = Exclude<
-  `${string}.${'js' | 'html' | 'css'}`,
-  DefaultEditorId
->;
+export type CustomEditorId = `${string}.${'js' | 'html' | 'css'}`;
```

Comment:
Note: I removed the `Exclude<>` because it wasn't working correctly. TS is smart enough to note files that end in the wrong suffix, but assigning `main.js` etc to a CustomEditorId compiled without warning. I was surprised too; it _looks_ like it should've worked.